### PR TITLE
Update VCR recording instructions for python auth tests

### PR DESCRIPTION
**Pull request by dimas-b**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1877)

### CODE REVIEW

File: python/tests/README.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
+     because the server will not recognize the auth tokens those tests use. In order to record cassettes for the
+     authentication tests:
+     1. Configure the Nessie server in dev mode with a real authentication provider (e.g. Keycloak)
+     1. Generate real auth tokens
```
Comment (snazy) [inline python/tests/README.md:17]: Can you add a short notice about how to generate one?

File: python/tests/README.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
...
+     1. Put real tokens into test sources
```
Comment (snazy) [inline python/tests/README.md:18]: Let's mention the parameter in application.properties.

Could we add one as a comment there?

File: python/tests/README.md (hunk 1 above)
Comment (dimas-b) [reply python/tests/README.md:17]: sure... updating...

Comment (dimas-b) [reply python/tests/README.md]: Added more specific instructions

File: python/tests/README.md (hunk 2 above)
Comment (dimas-b) [reply python/tests/README.md:18]: I'm not sure what you mean, sorry. The token is not meant to be in application.properties at all

Comment (snazy) [reply python/tests/README.md:18]: Strike my comment - it wasn't right.

File: python/tests/README.md [hunk 3]
```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
```
Comment (nastra) [inline python/tests/README.md:13]: same for `test_nessie_cli_auth.py`

File: python/tests/README.md [hunk 4]
```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
...
+     because the server will not recognize the auth tokens those tests use. In order to record cassettes for the
+     authentication tests:
+     1. Follow the instructions in the [Nessie Server README.md file](../../servers/quarkus-server/README.md) to start
+        the server in test mode with a real authentication provider and generate a real auth token.
+     1. Put real tokens into test sources
+     1. Run tests in recording mode
```
Comment (nastra) [inline python/tests/README.md:19]: Run auth tests in recording mode, such as `pytest --record-mode=all tests/test_nessie_cli_auth.py`

File: python/tests/README.md [hunk 5]
```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
+     because the server will not recognize the auth tokens those tests use. In order to record cassettes for the
+     authentication tests:
+     1. Follow the instructions in the [Nessie Server README.md file](../../servers/quarkus-server/README.md) to start
+        the server in test mode with a real authentication provider and generate a real auth token.
```
Comment (nastra) [inline python/tests/README.md:17]: would it make sense to mention `./mvnw quarkus:dev -am -pl :nessie-quarkus -Dquarkus.oidc.auth-server-url=http://localhost:8080/auth/realms/quarkus -Dnessie.server.authentication.enabled=true -Dquarkus.oidc.client-id=projectnessie` here?

Comment (dimas-b) [reply python/tests/README.md:17]: That's basically done under the `auth-test` quarkus profile, mentioned in the `quarkus-server/README.md`

Comment (dimas-b) [reply python/tests/README.md:17]: I'll be writing a more in-depth auth doc under #1981

File: python/tests/README.md (hunk 3 above)
Comment (dimas-b) [reply python/tests/README.md]: updated: a18b0aff916541eff14766b60b9dfdede0270d2f

File: python/tests/README.md (hunk 4 above)
Comment (dimas-b) [reply python/tests/README.md]: updated: a18b0aff916541eff14766b60b9dfdede0270d2f

**Review by nastra — APPROVED** [review]

**Pull request merged**